### DISCUSSION TITLE
Playground keeps fixing lint errors

### DISCUSSION THREAD

**Discussion by 6XGate** [typescript-eslint/typescript-eslint#7041] [Community Feedback]
Since the playground is suppose to demonstrate rule bugs with this plug-in, it seems really inappropriate for it to ever attempt to fix any lint error it finds. This is especially a major problem with false-positive bugs. This seems to mainly happen when pressing `Ctrl+S`.

**Comment by JoshuaKGoldberg** [comment]
> pressing <kbd>Ctrl+S</kbd>

Have you tried ... not doing that?

**Reply by 6XGate** [reply]
Hard habit to break :). Always making sure something is saved.

**Reply by bradzacher** [reply]
I didn't think Monaco captured the save combo in the browser - I thought it triggered the browser's save. Maybe they changed that behaviour in a recent version?

**Reply by JoshuaKGoldberg** [reply]
Yeah I never thought to check this 😄.

Because I keep having to recreate from scratch, here's a quicklink for a repro: https://typescript-eslint.io/play#ts=5.0.1-rc&sourceType=module&code=DYUwLgBAbghsCuIBcECCAndMCeAeAzmOgJYB2A5gHwDcQA&eslintrc=N4KABGBEBOCuA2BTAzpAXGUEKQAIBcBPABxQGNoBLY-AWhXkoDt8B6AQ2mncNqNPRREXAPbRI4MAF8QUoA&tsconfig=N4KABGBEDGD2C2AHAlgGwKYCcDyiAuysAdgM6QBcYoEEkJemy0eAcgK6qoDCAFutAGsylBm3TgwAXxCSgA

Now that we're on the topic, filed #7044 - I can see why this would be annoying!

**Reply by 6XGate** [reply]
Yeah, it surprised me too. Usually I would hit that combo and get a save dialog from the browser, but instead some auto-fixing.

**Reply by armano2** [reply]
> I didn't think Monaco captured the save combo in the browser - I thought it triggered the browser's save. Maybe they changed that behaviour in a recent version?

no it was added in #6529

> Yeah I never thought to check this 😄.

`ctrl/cmd + z` should be working fine in that scenario
